runtime-jvm/build.sbt

```diff
@@ -37,6 +37,7 @@ lazy val testSettings = Seq(
 
 lazy val root = project.in(file(".")).
   settings(name := "unison-runtime-root").
+  settings(test := (run in Test in main toTask "").value).
```

Comment:
Even better, wire this in `main` (and drop the `main` reference in the RHS), and the rest will be done by the aggregation.